composer.json

```diff
@@ -20,7 +20,7 @@
         "symfony/console": "~2.5",
         "symfony/process": "~2.5",
         "symfony/filesystem": "~2.5",
-        "henrikbjorn/lurker": "1.0.*@dev"
+        "henrikbjorn/lurker": "~1.1@dev"
```

Comment:
I would change it to `"flint/lurker": "~1.0"`. That should get you the 1.1 branch, and I think that version is more actively developed.